microconfig-cli/src/main/java/io/microconfig/MicroconfigMain.java

```diff
@@ -64,17 +69,19 @@ private void build() {
         }
     }
 
-    private void doBuild() {
-        enableLogger(!jsonOutput);
-
-        MicroconfigRunner runner = new MicroconfigRunner(rootDir, destinationDir);
-
-        Properties properties = runner.buildProperties(env, groups, services);
-        if (jsonOutput) {
-            out.println(toJson(properties.save(asConfigResult())));
-        } else {
-            properties.save(runner.toFiles());
-        }
+    private void doBuild(Set<String> envs) {
+        envs.forEach(env -> {
+            long startTime = currentTimeMillis();
+            MicroconfigRunner runner = new MicroconfigRunner(rootDir, new File(String.format("%s/%s", destinationDir, env)));
```

Comment:
The feature from the pr is pretty good. But  there is a small problem. The current implementation breaks the backward compatibility.  If someone updates microconfig version and generates configs, he will expect to have generated configs in the same folder that used to be before the update. But after the pr, configs will be generated to {env} folder. Probably its better to add a new cli parameter(envs). if you use envs param - configs will be generated to the ${env} dir. if you use the old cli env param - configs will be generated to the destination dir without {env}

Reply:
done !